costmanagementserv/src/main/java/com/costs/costmanagement/datamodels/ShowEpisodeCost.java

```diff
@@ -0,0 +1,55 @@
+package com.costs.costmanagement.datamodels;
+
+import javax.persistence.*;
+
+/*
+ This is the db model for SHOW_EPISODE_COSTS
+ */
+@Entity
+@Table(name = "SHOW_EPISODE_COSTS")
+public class ShowEpisodeCost {
```

Comment:
yeah, we should always have an id for each row in real life. the primary key is not useful here for our requirement but it will be useful if we want to do update on specific rows.

Reply:
added